File: src/main/java/com/zepben/cimbend/database/sqlite/tables/iec61968/assets/TableAssetOwners.java
/*
 * Copyright 2020 Zeppelin Bend Pty Ltd
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 */
package com.zepben.cimbend.database.sqlite.tables.iec61968.assets;

import com.zepben.annotations.EverythingIsNonnullByDefault;

@EverythingIsNonnullByDefault
public class TableAssetOwners extends TableAssetOrganisationRoles {

    @Override
    public String name() {
        return "asset_owners";
    }

    @Override
    protected Class<?> getTableClass() {
        return TableAssetOwners.class;
    }

    @Override
    protected Object getTableClassInstance() {
        return this;
    }

}

File: src/main/kotlin/com/zepben/evolve/database/sqlite/tables/iec61968/assets/TableAssetOwners.kt
/*
 * Copyright 2020 Zeppelin Bend Pty Ltd
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 */
package com.zepben.evolve.database.sqlite.tables.iec61968.assets

class TableAssetOwners : TableAssetOrganisationRoles() {

    override fun name(): String {
        return "asset_owners"
    }

    override val tableClass: Class<TableAssetOwners> = this.javaClass
    override val tableClassInstance: TableAssetOwners = this

}
